src/StdUtils.sol

```diff
@@ -7,7 +7,7 @@ abstract contract StdUtils {
     uint256 internal constant UINT256_MAX =
         115792089237316195423570985008687907853269984665640564039457584007913129639935;
 
-    function _bound(uint256 x, uint256 min, uint256 max) internal view virtual returns (uint256 result) {
+    function _bound(uint256 x, uint256 min, uint256 max) internal pure virtual returns (uint256 result) {
```

Comment:
```suggestion
    function _bound(uint256 x, uint256 min, uint256 max) private pure virtual returns (uint256 result) {
```

Reply:
I figured internal is preferable cause it also enables using `bound` without logging the result